modules/translation/i18n/i18n.go

```diff
@@ -141,3 +206,38 @@ func ResetDefaultLocales() {
 func Tr(lang, trKey string, trArgs ...interface{}) string {
 	return DefaultLocales.Tr(lang, trKey, trArgs...)
 }
+
+// TrOffset uses the default-locales to translate content to target language.
+// It uses the pre-computed translation keys->values.
+func TrOffset(offset int, trKey string, trArgs ...interface{}) string {
```

Comment:
Don't export this function. An offset is not something that makes sense outside of this specific implementation of a store.

Reply:
Has been removed by https://github.com/go-gitea/gitea/pull/20096/commits/8a1b976eb66a475afccd71b0a93237354099826a